packages/cashier/src/containers/cashier/cashier.jsx

```diff
@@ -123,7 +118,6 @@ const Cashier = ({
                         <VerticalTab
                             alignment='center'
                             id='cashier'
-                            side_note_class_name={getSideNoteClassName()}
```

Comment:
@hamid-deriv The issue was fixed [here](https://github.com/binary-com/deriv-app/pull/5929/commits/9f8404198aa92e06bc594888bf609ce7f18316ee), But I didn’t know if we can remove this `getSideNoteClassName`.
If we can remove it then the `side_note_class_name` prop on `VerticalTab` will be unused, We should also remove the prop in the component.

Reply:
@farzin-fs The `side_note_class_name` prop is used for the inner SideNote component in VerticalTab, so, I think it is safer to keep it, as VerticalTab is used in other packages like reports.

Reply:
@hamid-deriv The prop is unused 😬 I guess it was a patch for cashier and now we are removing the patch.

<img width="627" alt="image" src="https://user-images.githubusercontent.com/72082844/188567626-64fb4078-8745-42c0-82bb-3e849c2d7268.png">

Reply:
Removed :+1: